modules/nf-core/sentieon/applyvarcal/main.nf

```diff
@@ -22,19 +22,24 @@ process SENTIEON_APPLYVARCAL {
     task.ext.when == null || task.ext.when
 
     script:
-    def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args   = task.ext.args   ?: ''
+    def args2   = task.ext.args   ?: ''
```

Comment:
`args` and `args2` are both initialized from the same source (task.ext.args) and used twice in the script block. Is this correct?

Reply:
I think it should be `task.ext.args2`?

Reply:
Thanks for spotting

Reply:
```suggestion
    def args   = task.ext.args   ?: ''
    def args2  = task.ext.args2  ?: ''
```